tests/exporter/graph/test_graph_crawler.py

```diff
@@ -29,8 +29,8 @@ def test_generate_experiment_process_graph(self):
         experiment_graph = crawler.generate_experiment_graph(test_assay_process)
 
         # then
-        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 12)
-        self.assertEqual(len(experiment_graph.links.get_links()), 3)
+        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 15)
+        self.assertEqual(len(experiment_graph.links.get_links()), 4)
```

Comment:
analysis file, analysis protocol and analysis process was added in the nodes
and analysis process was added for the links